### PR TITLE
Adds Button To Clear Form

### PR DESCRIPTION
**Pull request by lqueipom**
Added:
- Function clearForm() to ratingScript.js to reset all form fields.
- Added clear button to course.html
- Close button to course.html placed in the div="retrieve-last-rating-message-container"

Modified:
- Made corresponding changes to JS file to reflect div tag added. 

Screenshot 2020-08-31 at 11 54 01 AM

### CODE REVIEW

File: src/main/webapp/course.html [hunk 1]
```diff
@@ -246,7 +246,37 @@ <h1 class="p-4 text-center">Add Your Review</h1>
...
+        <button
+          onclick="clearForm()"
+          id="clear-form"
+          type="button"
+          class="btn btn-lg btn-outline-success btn-block"
+          data-toggle="button"
+        >
+          Clear Form.
+        </button>
+        <!-- <div
```
Comment (hailed748) [inline src/main/webapp/course.html:259]: remove this commented code

File: src/main/webapp/course.html [hunk 2]
```diff
@@ -217,13 +217,13 @@ <h1 class="p-4 text-center">Add Your Review</h1>
             <div class="col">
               <p class="lead">Want to Translate to English?</p>
               <select class="custom-select mb-5" id="translate">
-                <option value="false" disabled selected>Wants to translate?</option>
+                <option value="false" disabled selected>
+                  Wants to translate?
```
Comment (ninaprabhu) [inline src/main/webapp/course.html:221]: Nit: Want to translate?

Comment (lqueipom) [reply src/main/webapp/course.html]: Good catch!

**Comment by LukeGMTravis** [comment]
Starting review

File: src/main/webapp/course.html [hunk 3]
```diff
@@ -236,17 +236,42 @@ <h1 class="p-4 text-center">Add Your Review</h1>
...
-        <div id="retrieve-last-rating-message" class="alert alert-light" role="alert></div>
+
+        <button
+          onclick="clearForm()"
+          id="clear-form"
+          type="button"
+          class="btn btn-lg btn-outline-success btn-block"
+          data-toggle="button"
+        >
+          Clear Form.
```
Comment (LukeGMTravis) [inline src/main/webapp/course.html:257]: Do you mean to have a period here?

Comment (lqueipom) [reply src/main/webapp/course.html]: Nop, thank you for pointing that out!

**Pull request merged**